benches/backtracking.rs

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use std::collections::BTreeMap;
+
+use criterion::*;
+use pubgrub::OfflineDependencyProvider;
+use version_ranges::Ranges;
+
+type Deps = BTreeMap<u32, BTreeMap<u8, BTreeMap<u32, Vec<(u8, u8)>>>>;
+
+fn bench(b: &mut Bencher, data: &str) {
+    let dependencies = ron::de::from_str::<Deps>(data).unwrap();
```

Comment:
Why does this use a different file format than the existing benchmark files?

Reply:
The new format is more compact to reduce the file size.